src/platform/plugins/shared/share/public/components/share_tabs.tsx

```diff
@@ -42,6 +42,9 @@ export const ShareMenuTabs = () => {
     tabs.push(embedTab);
   }
 
+  const showAccessModeContainer =
+    objectType === 'dashboard' && Boolean(sharingData?.accessModeContainer);
```

Comment:
We'd want to avoid having the share tab knowing about it's consumers when we can, they should only provide configuration. I think it would suffice for this iteration to check that should there be an `accessModeContainer`.